tests/emukit/test_acquisitions.py

```diff
@@ -98,6 +98,10 @@ def integrated_variance_acquisition(gpy_model, continuous_space):
 def squared_correlation_acquisition(vanilla_bq_model):
     return SquaredCorrelation(vanilla_bq_model)
 
+@pytest.fixture
+def probability_of_feasibility(gpy_model):
+    return ProbabilityOfFeasibility(gpy_model)
```

Comment:
I don't think we have this one merged yet, do we?

Reply:
That's weird - I'm not sure how that got there. I've removed it